### PR TITLE
feat: add set priority to workunit context menu

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

File: src/ecl/eclWatchTree.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import localize from "../util/localize";
 import { Item, Tree } from "./tree";
 import { eclWatchPanelView } from "./eclWatchPanelView";
 import { SaveData } from "./saveData";
+import { SMCServiceBase } from "@hpcc-js/comms/src/services/wsdl/WsSMC/v1.27/WsSMC";
```
Comment (GordonSmith) [inline src/ecl/eclWatchTree.ts:8]: Should be:
```ts
import { SMCService} from "@hpcc-js/comms";
```

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Needs rebase as well

**Review by GordonSmith — APPROVED** [review]
Ready to pull - can you squash the two commits.

**Comment by dehilsterlexis** [comment]
@GordonSmith need to test

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**